packages/manager/modules/octavia-load-balancer/src/create/constants.js

```diff
@@ -54,6 +54,9 @@ export const TRACKING_NAME =
 export const AGORA_ADDON_FAMILY = 'octavia-loadbalancer';
 
 export const SIZE_FLAVOUR_REGEX = /octavia-loadbalancer.loadbalancer-([sml]).hour.consumption/;
+export const AGORA_GATEWAY_REGEX = /gateway.s.hour.consumption/;
```

Comment:
**suggestion(nice-to-have):** The plancode is already saved in `packages/manager/modules/pci/src/projects/project/gateways/gateways.constants.js`. Do you think we can reuse it from here?